Libraries/Components/DatePicker/DatePickerIOS.ios.js

```diff
@@ -127,6 +132,7 @@ const DatePickerIOS = createReactClass({
           ref={ picker => { this._picker = picker; } }
           style={styles.datePickerIOS}
           date={props.date.getTime()}
+          locale={props.locale ? props.locale : undefined}
```

Comment:
Can we add something here which will take user locale by default ?

Note: no need to use ternary here, just use props.locale and add user locale in defaultProps.

Reply:
This is part of React Native and it's what is normally done for these props. If the locale isn't specified, it should use the default device locale.

Reply:
Awesome